src/modules/claim/resolvers/revoker.resolver.ts

```diff
@@ -1,8 +1,12 @@
 import { IRevokerDefinition } from '@energyweb/credential-governance';
```

Comment:
@nichonien can you please let me know if this PR implements the cache correctly?

Reply:
@whitneypurdum The approach for caching has changed recently, now we just inject in methods itself - https://github.com/energywebfoundation/ew-credentials/blob/98b4d13b3c29a1e3767d1e059431e8892efdbe64/packages/vc-verification/src/resolver/issuer-resolver.ts#L46.

So it's best to get the latest version.